src/msw/font.cpp

```diff
@@ -437,12 +465,7 @@ void wxNativeFontInfo::Init()
 
 int wxNativeFontInfo::GetPointSize() const
 {
-    // FIXME: using the screen here results in incorrect font size calculation
-    //        for printing!
-    const int ppInch = ::GetDeviceCaps(ScreenHDC(), LOGPIXELSY);
-
-    // BC++ 2007 doesn't provide abs(long) overload, hence the cast
-    return (int) (((72.0*abs((int)lf.lfHeight)) / (double) ppInch) + 0.5);
+    return lf.lfHeight;
```

Comment:
The changes here are very confusing because `LOGFONT::lfHeight` just is _not_ expressed in points.

I think we need to have the PPI inside `wxNativeFontItself` instead of having it (only) inside `wxFontRefData`, but then I don't really understand the semantics of font PPI yet...